hermes/hermes.ino

```diff
@@ -147,8 +148,17 @@ unsigned long lastSignificantMovementTime;
 // Initialization.
 void accelSetup() {
   Serial.println("BEGIN");
+
+  
+  // Try to initialise and warn if we couldn't detect the chip
+  digitalWrite(ONBOARD_LED_PIN, HIGH);
+  if (!lsm.begin())
+  {
+    Serial.println("Oops ... unable to initialize the LSM303. Check your wiring!");
+    while (1);
```

Comment:
do you want to stop here forever? maybe just having the shoe light up red isn't so bad?

Reply:
You are right. Makes more sense to go to red if the accel sensor is not working. However, at the moment only the RGB will show on the strip if this is the case. Red can be added in another branch.